Annotation/alternative_golden_set.json

```diff
@@ -657,7 +669,7 @@
                     }

```

Comment:
add "'s not yet" to str

Reply:
is it definitely an adj here and not just the past participle?